nameable.rb

```diff
@@ -0,0 +1,11 @@
+module Nameable
```

Comment:
- [ ] I suggest you make ```Nameable``` a class instead of a module, as it affects the proper functioning of the ```Person``` class